src/main/java/net/io/kino/model/EmailConfirmation.java

```diff
@@ -0,0 +1,55 @@
+package net.io.kino.model;
+
+import net.io.kino.KinoApplication;
+import org.springframework.boot.SpringApplication;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.mail.*;
```

Comment:
Musicie dodać javax.mail jako dependency w gradle. Nie wiem dlaczego to macie domyślnie, ale w openjdk tego nie ma